skrypty/command_sequence.lua

```diff
@@ -0,0 +1,34 @@
+CommandSequence = CommandSequence or {}
+
+function CommandSequence:new(...)
+    local o = {}
+    setmetatable(o, self)
+    self.__index = self
+    o.commands = table.n_flatten{...}
+    return o
+end
+
+function CommandSequence:add(command)
+    table.insert(self.commands, command)
+end
+
+function CommandSequence:get_next()
+    return table.remove(self.commands, 1)
+end
+
+function CommandSequence:send_next(silent)
+    local next = self:get_next()
+    if next then
+        expandAlias(next)
```

Comment:
Powinnismy chyba zaczac odchodzic od `expadAlias`:
https://wiki.mudlet.org/w/Manual:Miscellaneous_Functions#expandAlias

Reply:
W teorii tak, w praktyce nie wykonasz kodu wpisanego przez użytkownika inaczej.
Chodzi o to, że jak masz statyczne wywołanie innego alias, którego z góry znasz to tak. Tutaj jest to jedyny sposób, żeby wpisać do sekwencji dowolny alias.

Reply:
Więc zakładam, że po prostu z głową :)